packages/network/src/types.js

```diff
@@ -78,11 +78,16 @@ export {};
  *   listener has been removed
  */
 
+/**
+ * @typedef {object} SendOptions
+ * @property {bigint} [relativeTimeoutNs] timeout the packet if an ack is not received by this time
+ */
+
```

Comment:
What I meant to say was something like:

```suggestion
/**
 * @typedef {object} WellKnownSendOptions
 * @property {bigint} [relativeTimeoutNs] timeout the packet if an ack is not received by this time
 */

/**
 * @typedef {WellKnownSendOptions & Record<string, any>} SendOptions
 */

```